packages/@lwc/node-reactions/src/global/README.md

```diff
@@ -0,0 +1,2 @@
+Note:
+Modules in this folder contain stateful information. This is important when working with unit test environments. Jest for example shares DOM prototypes across test suites. Care must be taken to avoid repeatedly patching prototypes.
```

Comment:
The readme is not applicable anymore. We were trying to use a Symbol to cache the init routine on the DocumentPrototype. That was causing some issues with JEST. We have switched to a string field since.

Reply:
Agreed. This is a common problem. May be we can reset the prototypes at the root Jest config. Resetting at the end of each test suite will become tedious and easy to forget.